tools/benchmark/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## 0.51.0
 
+Provide @benchmarkCustom feature
```

Comment:
We should add more detail here. Detailing what `@benchmarkCustom` is and how to use it.

We should probably also update the package's README to include details about this new feature.

Reply:
updated